content/docs/deployment/csmoperator/_index.md

```diff
@@ -0,0 +1,228 @@
+---
```

Comment:
Generic comment across the board - Please remove EMC from the docs.

Reply:
Removed references to EMC